### ISSUE TITLE
UnicodeDecodeError

### ISSUE THREAD

**Issue by gnzlbg** [dtolnay/cargo-expand#17]
cmd: `cargo expand --color never`
version: v0.3.10

Error:

```shell
** Error while highlighting:
UnicodeDecodeError: 'ascii' codec can't decode byte 0xe2 in position 1832403: ordinal not in range(128)
   (file "/usr/local/lib/python2.7/site-packages/Pygments-1.6-py2.7.egg/pygments/lexer.py", line 168, in get_tokens)
```

I don't understand why pygments is invoked at all when using `--color never`. I can also reproduce without the option.

**Comment by gnzlbg** [comment]
@dtolnay I get the same error when I do `cargo expand` (without `--color never`).

**Comment by dtolnay** [comment]
Can you share the code of the crate you are expanding?

Do you get the same error piping to pygmentize manually?

```bash
cargo rustc --color=always -- -Zunstable-options --pretty=expanded | pygmentize -l rust
```

Can you tell what the line containing position 1832403 (I assume a byte offset from the beginning of the expanded output) looks like?

**Comment by gnzlbg** [comment]
> Can you share the code of the crate you are expanding?

I am expanding the `stdsimd` crate. 

> Do you get the same error piping to pygmentize manually?

Yes.

> Can you tell what the line containing position 1832403

```shell

        /// # let mut some_utf16_words = [0u16; 8];
        /// # let mut more_utf16_words = [0u16; 8];
        /// # '❤'.encode_utf16(&mut some_utf16_words);
        /// # '𝕊'.encode_utf16(&mut more_utf16_words);
        /// // Load the input
        /// let a = __m128i::from(u16x8::load(&some_utf16_words, 0));

```

**Comment by dtolnay** [comment]
Thanks! I have not been able to reproduce the error but we are on the right track. Does it succeed with `-O encoding=utf8`?

```bash
cargo rustc --color=always -- -Zunstable-options --pretty=expanded | pygmentize -l rust -O encoding=utf8
```

**Comment by gnzlbg** [comment]
> Does it succeed with -O encoding=utf8?

Yes! That works!

**Comment by gnzlbg** [comment]
I am on MacOS X btw, maybe it has to do with the version of python installed here.

**Comment by dtolnay** [comment]
I published cargo-expand 0.3.12 that always passes `-O encoding=utf8` to pygmentize.

**Comment by gnzlbg** [comment]
Thanks! It works now!
